### ISSUE TITLE
[Error]: 404: Not Found

### ISSUE THREAD

**Issue by quamt** [anidl/multi-downloader-nx#703]
### Program version

5.0.3

### Operating System

Windows

### Type

GUI

### Service

Crunchyroll

### Command used

NIL

### Show ID

ALL

### Episode

ALL

### Console Output

```Shell
Chapter request successful
[ERROR] 404: Not Found
[ERROR] {"code":"content.get_video_streams_v2.resource_not_found","context":[]}
[ERROR] Request Stream URLs FAILED! Attempting fallback
Selecting raw stream
```

### Additional Information

I'm not sure if this is an issue on my backend, but the message has popped up since today.
Using ` none DRM` server.
It happens with everything on `Crunchy`.

**Comment by Jaynator495** [comment]
This message is just a warning, it really shouldn't say error. Trying to guess the ID for the endpoint isn't always right, so sometimes it has to use a fallback. This is specifically an issue for series without version information in the API

**Comment by quamt** [comment]
It continues to download, correct.
Yes. the ERROR made me think something was wrong.
If this is intended behaviour this can be closed. 
Thank you.

**Comment by Jaynator495** [comment]
Yes this is intended, though I'll look into changing it to a warning
